forms-web-app/src/controllers/appellant-submission/appeal-statement.js

```diff
@@ -14,7 +14,12 @@ exports.postAppealStatement = async (req, res) => {
 
   const appeal = {
     ...req.session.appeal,
-    'appeal-statement': req.files && req.files['appeal-statement'],
+    'appeal-upload':
+      req.files &&
+      req.files['appeal-upload'] &&
+      JSON.stringify({
+        name: req.files['appeal-upload'].name,
+      }),
```

Comment:
we're going to need one of these types of checks & conversions for every file used in the form. 

The upstream API expects a String, so I have stringify here which is then passed as a string in JSON encoded body for `createOrUpdateAppeal` - not a fan of the double encoding, but I can live with this bit.